src/lib.rs

```diff
@@ -2,8 +2,8 @@
 //!
 //! The driver must be initialised by passing an I2C or SPI interface peripheral to the [`Builder`],
 //! which will in turn create a driver instance in a particular mode. By default, the builder
-//! returns a [`RawMode`] instance which isn't very useful by itself. You can coerce the driver
-//! into a more useful mode by calling `into()` and defining the type you want to coerce to. For
+//! returns a [`DisplayProperties`] instance which is a low level interface to manipulate the display properties (e.g. rotation).
+//! You can coerce the driver into a more useful mode by calling `into()` and defining the type you want to coerce to. For
```

Comment:
Please wrap this to 100 columns.

While we're here, I'd also suggest rewording the `You can coerce the driver` part to `The driver can be coerced`.